templates/_layouts/_root.html

```diff
@@ -0,0 +1,129 @@
+  <meta name="description" content="Ubuntu brand, app and web guidelines that help you create professional materials, software, sites, apps that build the Ubuntu brand.">
+  <meta name="keywords" content="Ubuntu, Design, Brand guidelines, web guidelines, apps guidelines, colors, buttons, layout, styleguide" />
+  <meta name="author" content="Canonical" />
+
+  <!-- Google Tag Manager -->
+  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+  new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+  j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+  'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+  })(window,document,'script','dataLayer','GTM-NV2KV6F');</script>
+  <!-- End Google Tag Manager -->
+</head>
+
+
+<body>
+
+<!-- Google Tag Manager (noscript) -->
+<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-NV2KV6F"
+height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+<!-- End Google Tag Manager (noscript) -->
+
+
+<a href="#main-content" class="p-link--skip">Jump to main content</a>
+
+<header id="navigation" class="p-navigation is-dark">
+  <div class="p-navigation__row">
+    <div class="p-navigation__banner">
+      <div class="p-navigation__tagged-logo">
+        <a class="p-navigation__link" href="/">
+          <div class="p-navigation__logo-tag">
+            <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="">
+          </div>
+          <span class="p-navigation__logo-title">Design</span>
+        </a>
+      </div>
+      <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
+      <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
+    </div>
+    <nav class="p-navigation__nav">
+      <ul class="p-navigation__items">
+        <li class="p-navigation__item {% if path.startswith('/brand') %}is-selected{% endif %}"><a class="p-navigation__link " href="/brand/">Brand</a></li>
+        <li class="p-navigation__item {% if path.startswith('/font') %}is-selected{% endif %}"><a class="p-navigation__link" href="/font/">Font</a></li>
+        <li class="p-navigation__item {% if path.startswith('/web') %}is-selected{% endif %}"><a class="p-navigation__link" href="/web/">Web</a></li>
+        <li class="p-navigation__item {% if path.startswith('/downloads') %}is-selected{% endif %}"><a class="p-navigation__link" href="/downloads/">Downloads</a></li>
+      </ul>
+      <ul class="p-navigation__items global-nav"></ul>
+    </nav>
+  </div>
+</header>
+
+
+{% block body %}{% endblock %}
+
+
+<footer class="p-strip" style="border-top: 1px solid #cdcdcd;">
+  <div class="row">
+    <div class="col-8">
+      <p class="u-no-max-width">&copy; 2019 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical&nbsp;Ltd.</p>
```

Comment:
Maybe worth updating to 2023 at this point :D 
```suggestion
      <p class="u-no-max-width">&copy; 2023 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical&nbsp;Ltd.</p>
```

Reply:
good idea.